internal/librarian/generate_test.go

```diff
@@ -404,10 +483,11 @@ func TestGenerateRun(t *testing.T) {
 					},
 				},
 			},
-			container:         &mockContainerClient{},
-			build:             true,
-			wantGenerateCalls: 1,
-			wantBuildCalls:    1,
+			container:          &mockContainerClient{},
+			build:              true,
+			wantGenerateCalls:  1,
+			wantBuildCalls:     1,
+			wantConfigureCalls: 1,
```

Comment:
The test title is `regeneration of API`, why the `configure` command is invoked?